fairytale.md

```diff
@@ -3,6 +3,8 @@
 ------------------------------------
 
 
+The interdimensional journey begins to the best of our knowledge in the year 30 AD with the execution of Jesus of Nazareth by Pontius Pilate. The journey is that of mystery and great confusion for our characters in 1930s Moscow 
```

Comment:
Hi, thanks for the merge but I do not think this should be the first line. It just makes no sense with the whole fairytale. I think this should be moved to line 12 to have a little connection with the rest.
Please modify the merged document. If you cannot manage by October 22 at 20.00 please let me know.